### ISSUE TITLE
Not working with independent version strategy of Lerna

### ISSUE THREAD

**Issue by softvar** [ajaymathur/popular-package#7]
Error:
```
Calculating internal dependency DataError: Invalid version: "independent"
```

lerna.json

```
{
  "lerna": "2.6.0",
  "version": "independent",
  "commands": {
    "publish": {
      "message": "chore(release): publish %s",
      "allowBranch": "master",
      "ignore": []
    }
  },
  "packages": ["packages/@myworld/*"],
  "npmClient": "yarn"
}

```

**Comment by ajaymathur** [comment]
Thanks! 😄 

I used [read-pkg](https://github.com/sindresorhus/read-pkg]) to read lerna json, (wrong approach), so it validated json and gave error.

https://github.com/ajaymathur/popular-package/blob/master/lib/utils/dir.js#L11

**Comment by ajaymathur** [comment]
@softvar fixed in popular-package@0.3.1 🙂

**Comment by softvar** [comment]
Works now. 
That was fast! Thanks! 👍
